src/CalendarComponent.jsx

```diff
@@ -22,18 +21,25 @@ class CalendarComponent extends React.Component {
 	* @description init the js date flatpickr component
 	*/
 	componentDidMount() {
-		const options = {
-			onChange: this.onChange,
-			onOpen: this.onOpen,
-			onClose: this.onClose,
-			altInput: true,
-			allowInput: true,
-			altFormat: 'D M d, Y', // TODO localize
-			defaultDate: this.props.value
-		};
+		// flatpickr uses `window` on import,
+		// which breaks on server-sider render.
+		// lazy-loading flatpickr ensures it is
+		// imported only in clientside envs.
+		const Flatpickr = require('flatpickr');
```

Comment:
I originally had it as an `import` but was getting a syntax error on compile because of the top-level specification. strangely, it was compiling in mup-web but didn't in here.